installing/installing_aws/installing-restricted-networks-aws.adoc

```diff
@@ -22,12 +22,12 @@ according to your company's policies.
 
 .Prerequisites
 
-* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry on your bastion host]
+* xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[Create a mirror registry on your mirror host]
  and obtain the `imageContentSources` data for your version of {product-title}.
 +
 [IMPORTANT]
 ====
-Because the installation media is on the bastion host, use that computer
+Because the installation media is on the mirror host, you may use that computer
```

Comment:
I get that you're trying to step back from the bastion, but do you have a particular workflow in mind? "May implies permission, and that's not really what we mean here.

I'd either ~you may~ or, if you want to back it off more and still want them to use the installer that they generated from the mirror repo, say something like "Because the installation media is on the mirror host, the following steps assume that you use that computer to complete all installation steps. If you cannot complete the {product-title} installation from the mirror host, use the method that best fits your restrictions to ensure that you use the installation program that you generated from the mirrored content."